README.md

```diff
@@ -7,8 +7,8 @@ The libraries are still in active development, and new features will incremental
 ## Libraries
 
 - [nns](/packages/nns): interfacing with the **ledger** and **governance** canisters of the IC
-- sns-js: interacting with a Sns project (_coming soon_)
-- cmc-js: querying the Cmc canister (_coming soon_)
+- [sns](/packages/sns): interacting with a sns project (_coming soon_)
+- cmc: querying the Cmc canister (_coming soon_)
 
 ## Integration with NNS-dapp
 
```

Comment:
Can you add here or in ahoter readme the comments you did about the candid files as well:

```
a. something.did -> the candid definition
b. something.d.ts -> the typescript definition for the types and service of the did files (generated with didc)
c. something.idl.js -> the factory js file (generated with didc)
d. something.idl.d.ts -> the typescript definition of the above factory file
e. something.certified.idl.js -> the factory js file for the certified calls (generated with didc)
f. something.certified.idl.d.ts -> the typescript definition of the above factory file
```

That is very helpful.

Reply:
sure done, I included, extended and refactored the `HOW-To.md` to `HACKING.md`.

i gonna merge the PR to make/test these changes available for nightly-build but if you notice anything related that should be improved, let me know. happy to provide another PR.